doc/api/date/date-formatter.md

```diff
@@ -168,6 +168,17 @@ Use skeletons for more flexibility (see its description [above](#parameters)).
 | `"yQQQ"` | `"Q4 2010"` |
 | `"yQQQQ"` | `"4th quarter 2010"` |
 
+Use a date fields part and a time fields part (in Canonical Order) to form skeleton for a custom Date Time format.
+
+| `skeleton` | `Globalize( "en" ).dateFormatter( skeleton )( new Date( 2010, 10, 1, 17, 55 ) )` |
+| --- | --- |
+| `"GyMMMEdhms"` | `"Mon, Nov 1, 2010 AD, 5:55:00 PM"` |
+| `"Ehms"` | `"Mon 5:55:00 PM"` |
+| `"yQQQHm"` | `"Q4 2010, 17:55"` |
+| `"MMMEdhm"` | `"Mon, Nov 1, 5:55 PM"` |
+| `"yMMMdhm"` | `"Nov 1, 2010, 5:55 PM"` |
+
+
```

Comment:
The fact there are date and field parts in the same skeleton should be transparent for the end-user. Let's have your examples merged in the above table. Also, feel free to update the previous paragraph to include "(in Canonical Order)".